doc/classes/EditorSceneFormatImporter.xml

```diff
@@ -44,17 +44,17 @@
 		</method>
 	</methods>
 	<constants>
-		<constant name="IMPORT_SCENE" value="1">
+		<constant name="IMPORT_SCENE" value="1" enum="ImportFlags">
 		</constant>
-		<constant name="IMPORT_ANIMATION" value="2">
+		<constant name="IMPORT_ANIMATION" value="2" enum="ImportFlags">
```

Comment:
In the near future, to solve https://github.com/godotengine/godot-proposals/issues/7494, I was thinking of refactoring these into an enum with many more options (scene, animation, node, mesh, material, etc). These would be a sequential enum (0, 1, 2, 3, 4) separate from the flags.

This doesn't technically block this change because we likely want to keep these values around anyway for compat, but making them a part of the enum seems like the wrong direction.

Reply:
They are defined as enums in the source code, though.